src/Amadeus.Server/Controllers/UserController.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amadeus.Server.Models;
+using Amadeus.Server.Services;
+using Amadeus.Server.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Amadeus.Server.Controllers
+{
+	/// <summary>
+	/// The controller for user related interactions.
+	/// </summary>
+	public class UserController
```

Comment:
Don't create a controller to wrap a repository. This is usless.

Reply:
Worthless little creature